addons/mail/static/src/messaging/component/chatter/chatter_tests.js

```diff
@@ -55,12 +55,12 @@ QUnit.module('Chatter', {
 QUnit.test('base rendering when chatter has no attachment', async function (assert) {
     assert.expect(6);
 
-    const messages = [...Array(60).keys()].map(id => {
+    const messages = [...Array(60).keys()].map(i => {
         return {
             author_id: [10, "Demo User"],
-            body: `<p>Message ${id}</p>`,
+            body: `<p>Message ${60-i}</p>`,
             date: "2019-04-20 10:00:00",
-            id,
+            id: 60-i,
```

Comment:
changes look useless

Reply:
It's made so the chatter is shown correctly as it's loading messages "most recent first"

Reply:
I fail to see how it fixes that.
Before change list contains message with ids 0...59 (using id 0 is no safe though), and now it contains message with ids 60...1.
The order in records should not affect mocked RPCs.
Each of them should explicitly sort them, i.e. by ids.

Reply:
I'll check because I think it's not ordered by the mocked RPCs but I'm not sure anymore

Reply:
It was working, don't know why I thought it wasn't the case

Reply:
I would still change it a bit just to make sure no ID has 0 if that is the case currently.

Reply:
Still have to push the diff but that's what I did :)